packages/headless/src/controllers/commerce/context/cart/headless-cart.test.ts

```diff
@@ -117,7 +119,7 @@ describe('headless commerce cart', () => {
       jest.resetAllMocks();
     });
 
-    it('emits the #ec.purchase event with the expected payload', () => {
+    it.skip('emits the #ec.purchase event with the expected payload', () => {
```

Comment:
🧐

Reply:
Yeah ain't complete chief. But here, we ain't gonna emit the ecPurchase as we mock the action.
I need to UT my action instead.